<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminUserRequest</class>
 <widget class="QMainWindow" name="AdminUserRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border:none;
background-color:transparent;
color:#fff;
}



#label
{
color:#fff;

}

#label_2
{
color:#071e26;

}

#centralwidget{
	
	background-color: rgb(75, 75, 75);
}

#sidemenu{
background-color: rgb(171, 171, 171);
border-radius:20px;
}

QPushButton{
padding: 10px;
background-color: rgb(75, 75, 75);
border-radius: 5px;

}

#menuButton{
	background-color: rgb(171, 171, 171);
padding: 10px;
border-radius: 5px;

}

#mainbody{
background-color:071e26;
border-radius:10px;
}

#widget_4{
background-color: rgb(171, 171, 171);
border-radius:20px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>My Desktop Application</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1080</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="mainbody">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1050</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFrame" name="widget_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>490</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="menu_btn_users_request">
               <property name="text">
                <string>Approved User Request</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/users.svg</normaloff>:/icons/users.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="menu_btn_upload_document">
               <property name="text">
                <string>Upload Document</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/upload.svg</normaloff>:/icons/upload.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="sidemenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>490</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>2000</width>
              <height>490</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QTableWidget" name="tableWidget">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>460</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QTableWidget {
        background-color: rgb(171, 171, 171);
        alternate-background-color: #e6e6e6;
        selection-background-color: #a6a6a6;
        font-size: 14px;
    }
    
    QTableWidget QHeaderView::section {
        background-color: rgb(171, 171, 171);
        color: black;
        font-weight: bold;
    }
    
    QTableWidget::item:selected {
        color: white;
    }</string>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Designation</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Email</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Approve</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_4">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>480</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_5">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>480</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <widget class="QPushButton" name="teachersViewBtn">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>10</y>
                     <width>83</width>
                     <height>36</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>Teachers</string>
                   </property>
                  </widget>
                  <widget class="QPushButton" name="studentsViewBtn">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>60</y>
                     <width>83</width>
                     <height>36</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>Students</string>
                   </property>
                  </widget>
                  <widget class="QPushButton" name="ApproveBtn">
                   <property name="geometry">
                    <rect>
                     <x>10</x>
                     <y>420</y>
                     <width>83</width>
                     <height>36</height>
                    </rect>
                   </property>
                   <property name="text">
                    <string>Approve</string>
                   </property>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
